packages/fiori-elements-writer/src/index.ts

```diff
@@ -222,7 +222,9 @@ async function generate<T extends {}>(
             {
                 htmlTarget: feApp.appOptions?.generateIndex
                     ? 'index.html'
-                    : `test/flpSandbox.html?sap-ui-xx-viewCache=false#${feApp.app.flpAppId}`
+                    : `test/${
```

Comment:
Nested ternary, thought this would be flagged?

Reply:
Updates made to reduce complexity below 15